rebar.config

```diff
@@ -29,12 +29,16 @@
    {branch, "master"}}},
   {katja, ".*",
    {git, "git://github.com/nifoc/katja.git",
-   {branch, "master"}}}
+   {branch, "master"}}},
+  {rebar_lock_deps_plugin, ".*", {git, "git://github.com/seth/rebar_lock_deps_plugin.git", {tag, "3.1.0"}}}
  ]}.
 
 {erl_opts, [{src_dirs, [src]},
            {parse_transform, lager_transform}]}.
 
+{plugin_dir, ".plugins"}.
```

Comment:
There is no directory named `.plugins`. in git. Is this config needed? or generated?